<!DOCTYPE html>

<html>

<head>

    <title>Bo Li's Home Page</title>

    <style type="text/css">
        #portrait {
            width: 20%;
        }

        #address {
            float: right;
            margin-right: 60%;
            margin-top: 2%;
        }

        #clearit {
            clear: right;
        }

        #research {
            margin-left: 2.5em;
        }
    </style>

    <script>
        (function (i, s, o, g, r, a, m) {
        i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
            (i[r].q = i[r].q || []).push(arguments)
        }, i[r].l = 1 * new Date(); a = s.createElement(o),
            m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
        })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');

        ga('create', 'UA-68829914-1', 'auto');
        ga('send', 'pageview');

    </script>

</head>


<body>

    <img src="BoLi_portrait.jpg" alt="Bo Li's portrait" id="portrait" />

    <div id="address">
        <h1>Bo Li</h1>

        <p> Postdoctoral Associate <br />
            Broad Institute of MIT and Harvard </p>

        <p> 415 Main Street <br />
            Cambridge, MA 02142, USA </p>

        <p><a href="Curriculum_vitae.pdf"
                onclick="ga('send', 'event', { eventCategory: 'download', eventAction: 'click', eventLabel: 'CV', nonInteraction: true });">
                Curriculum Vitae </a></p>

        <p> Email: libo [at] broadinstitute [dot] org </p>

    </div>

    <h2 id="clearit"> Academic Employment </h2>

    <ul>
        <li> 2017.07 &mdash; present &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp Postdoctoral associate at <a
                href="https://www.broadinstitute.org/regev-lab">Regev Lab</a>, Broad Institute of MIT and Harvard </li>
        <li> 2013.08 &mdash; 2017.06 &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp Postdoctoral researcher at <a
                href="http://pachterlab.github.io">Pachter Lab</a>, University of California at Berkeley </li>
    </ul>

    <h2> Education </h2>

    <ul>
        <li> 2008.09 &mdash; 2013.07 &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp Ph.D. in Computer Sciences, &nbsp University of
            Wisconsin-Madison </li>
        <li> 2008.09 &mdash; 2010.05 &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp M.S. in Computer Sciences, &nbsp University of
            Wisconsin-Madison </li>
        <li> 2004.09 &mdash; 2008.07 &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp B.E. in Computer Science and Engineering
            (<strong>ACM Honor Class</strong>), &nbsp Shanghai Jiao Tong University </li>
    </ul>

    <h2> Research </h2>

    <p id="research"> I'm a computational biologist working on single cell RNA-Seq data analysis. I use statistics and
        machine learning extensively in my research.</p>

    <h2> Publications </h2>
    <p> * Equal contributors </p>

    <ul>
        <li>Gaublomme JT*, <strong>Li B</strong>*, McCabe C, Knecht A, Drokhlyansky E, Van Wittenberghe N, Waldman J,
            Dionne D, Nguyen L, De Jager P, Yeung B, Zhao X, Habib N, Rozenblatt-Rosen O and Regev A. (2018). <a
                href="https://www.biorxiv.org/content/early/2018/11/23/476036"><strong>Nuclei multiplexing with barcoded
                    antibodies for single-nucleus genomics</strong></a>. <em>bioRxiv</em>, 10.1101/476036.</li>
        <br />
        <li><strong>Li B</strong>, Tambe A, Aviran S and Pachter L. (2017). <a
                href="https://www.cell.com/cell-systems/abstract/S2405-4712(17)30139-4"><strong>PROBer provides a
                    general toolkit for analyzing sequencing-based toeprinting assays</strong></a>. <em>Cell
                Systems</em>, <strong>4</strong>(5), 568-574.</li>
        <br />
        <li>Choudhary K, Shih NP, Deng F, Ledda M, <strong>Li B</strong> and Aviran S. (2016). <a
                href="http://bioinformatics.oxfordjournals.org/content/early/2016/09/02/bioinformatics.btw501.long"><strong>Metrics
                    for rapid quality control in RNA structure probing experiments</strong></a>.
            <em>Bioinformatics</em>, <strong>32</strong>(23), 3575-3583.</li>
        <br />
        <li>Zeng X, <strong>Li B</strong>, Welch R, Rojo C, Zheng Y, Dewey CN and Kele&#642; S. (2015). <a
                href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004491"><strong>Perm-seq:
                    Mapping protein-DNA interactions in segmental duplication and highly repetitive regions of genomes
                    with prior-enhanced read mapping</strong></a>. <em>PLoS Computational Biology</em>,
            <strong>11</strong>(10):e1004491.</li>
        <br />
        <li><strong>Li B</strong>*, Fillmore N*, Bai Y, Collins M, Thomson JA, Stewart R and Dewey CN. (2014). <a
                href="http://genomebiology.com/2014/15/12/553/abstract"><strong>Evaluation of <i>de novo</i>
                    transcriptome assemblies from RNA-Seq data</strong></a>. <em>Genome Biology</em>,
            <strong>15</strong>(12), 553. <a href="http://www.biomedcentral.com/info/about/mostviewed"><img
                    src="highly-accessed.gif" alt="Highly accessed"></a> </li>
        <br />
        <li>Haas B, Papanicolaou A, Yassour M, Grabherr M, Blood PD, Bowden J, Couger MB, Eccles D, <strong>Li
                B</strong>, Lieber M, MacManes MD, Ott M, Orvis J, Pochet N, Strozzi F, Weeks N, Westerman R, William T,
            Dewey CN, Henschel R, LeDuc RD, Friedman N and Regev A. (2013). <a
                href="http://www.nature.com/nprot/journal/v8/n8/full/nprot.2013.084.html"><strong><i>De novo</i>
                    transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation
                    and analysis</strong></a>. <em>Nature Protocols</em>, <strong>8</strong>(8), 1494-1512. </li>
        <br />
        <li> <strong>Li B</strong> and Dewey CN. (2011). <a
                href="http://www.biomedcentral.com/1471-2105/12/323"><strong>RSEM: accurate transcript quantification
                    from RNA-Seq data with or without a reference genome</strong></a>. <em>BMC Bioinformatics</em>.
            <strong>12</strong>:323. <a href="http://www.biomedcentral.com/info/about/mostviewed"><img
                    src="highly-accessed.gif" alt="Highly accessed"></a> </li>
        <br />
        <li> Chung D, Kuan PF, <strong>Li B</strong>, Sanalkumar R, Liang K, Bresnick E, Dewey CN and Kele&#642; S.
            (2011). <a
                href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002111"><strong>Discovering
                    transcription factor binding sites in highly repetitive regions of genomes with multi-read analysis
                    of ChIP-seq data</strong></a>. <em>PLoS Computational Biology</em>, 7:e1002111. </li>
        <br />
        <li> <strong>Li B</strong>, Ruotti V, Stewart RM, Thomson JA and Dewey CN. (2010). <a
                href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/26/4/493"><strong>RNA-Seq gene
                    expression estimation with read mapping uncertainty</strong></a>. <em>Bioinformatics</em>,
            <strong>26</strong>(4), 493-500. </li>
    </ul>

    <h2> Software </h2>
    <ul>
        <li><a href="http://pachterlab.github.io/PROBer">PROBer</a>: A principled and unified probabilistic framework
            for analyzing sequencing-based 'toeprinting' assays.</li>
        <br />
        <li><a href="http://deweylab.biostat.wisc.edu/detonate">DETONATE</a>(<i><b>DE</b> novo</i>
            <b>T</b>ranscript<b>O</b>me r<b>N</b>a-seq <b>A</b>ssembly with or without the <b>T</b>ruth
            <b>E</b>valuation): A <i>de novo</i> transcriptome assembly evaluation package, which contains two
            components, RSEM-EVAL and REF-EVAL. I developed the RSEM-EVAL component.</li>
        <br />
        <li><a href="http://deweylab.biostat.wisc.edu/rsem">RSEM</a>(<b>R</b>NA-<b>S</b>eq by
            <b>E</b>xpectation-<b>M</b>aximization): Accurate gene and isoform expression levels quantification from
            RNA-Seq data with or without a reference genome.</li>
        <br />
        <li><a href="http://deweylab.biostat.wisc.edu/csem">CSEM</a>(<b>C</b>hIP-<b>S</b>eq multi-read allocation using
            <b>E</b>xpectation-<b>M</b>aximization): One of the first ChIP-Seq multi-mapping read allocators. CSEM
            allows multi-reads to be utilized by peak callers.</li>
        <br />
    </ul>

    <h2> Miscellaneous </h2>
    <ul>
        <li><a href="https://manubot.org/" rel="nofollow">Manubot</a>: a piece of open-source software that automates the process of collating, formatting and publishing a scientific paper. Each time an author creates a version of the manuscript, the software logs that event on the Bitcoin blockchain.</li>
        <br />
        <li><a href="https://www.sciencematters.io/" rel="nofollow">Sciencematters EUREKA</a>: Using a peer-review platform called Eureka, ScienceMatters offering its triple-blind peer-review process through a publishing platform that uses the Ethereum blockchain (as well as the prevailing <a href="https://coindataflow.com/en">many other</a> projects).</li>
        <br />
        <li><a href="https://artifacts.ai/" rel="nofollow">ARTiFACTS</a>: provides a forum in which researchers can upload almost anything that they deem worth sharing, with each file logged to a blockchain. </li>
        <br />

    </ul>
</body>

</html>